README.md

```diff
@@ -1,2 +1,54 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # capstone-1
 Building an online website for a conference
+
+![screenshot](./images/2021-10-12.png)
+
+.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Application Page url
+
+https://phelian23.github.io/capstone-1/
+
```

Comment:
- [ ] Also as requirements the video is mandatory for this capstone project, please recorded at least 5 min video.

![lll](https://user-images.githubusercontent.com/61562626/137153577-a8a2aa6a-6e41-4520-9a85-df88e572bc50.jpeg)